example/shoot-cert-service/base/doc.go

```diff
@@ -0,0 +1,7 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//go:generate extension-generator --name=extension-shoot-cert-service --provider-type=shoot-cert-service --component-category=extension --extension-oci-repository=local-skaffold/gardener-extension-shoot-cert-service/charts/extension:v0.0.0 --destination=$PWD/extension.yaml
+
```

Comment:
I think the Kustomize call is missed here to save final example resource somewhere?!

Reply:
Added example extension resource